src/test/scala/ing/wbaa/druid/DruidClientSpec.scala

```diff
@@ -17,14 +17,16 @@
 
 package ing.wbaa.druid
 
-import akka.http.scaladsl.model.StatusCode
+import akka.http.scaladsl.model.headers.RawHeader
+import akka.http.scaladsl.model.{ HttpProtocols, StatusCodes }
 import ing.wbaa.druid.client.{ DruidHttpClient, HttpStatusException }
 import ing.wbaa.druid.definitions.{ CountAggregation, GranularityType }
 import org.scalatest._
 import org.scalatest.concurrent._
 
 import scala.concurrent.duration._
 import scala.language.postfixOps
+import scala.util.Success
```

Comment:
unused import statement

Reply:
Fixed in bf0dae9e638c277aa6b9c05c9bc4ac6c83ecc262